content/tutorials/token-authentication.textile

```diff
@@ -997,7 +997,7 @@ blang[nodejs,ruby,python,java].
   ```
 
 blang[php].
-  Time to get started. First you'll need to add two panels, one that contains a simple login form, and the other with a logout link to the HTML page. Add the following beneath the @</h1>@ tag in your @index.php@ file:
+  Time to get started. Firstly you will need to add two panels, one that contains a simple login form, and the other with a logout link to the HTML page. Add the following beneath the @</h1>@ tag in your @index.php@ file:
```

Comment:
'Firstly, you will need to add' - inconsistent with the versions in other languages.